test/config/credentials_default.json

```diff
@@ -19,8 +20,8 @@
     ],
     "ora": [
       {
-        "username": "vagrant",
-        "password": "vagrant"
+        "username": "onrack",
+        "password": "onrack"
```

Comment:
The ora credentials are onrack/onrack in the MN config that we use. Remember this is the public config set.